web-frontend/cypress/e2e/acceptance/home/send.cy.tsx

```diff
@@ -54,47 +113,197 @@ describe('E2E | Acceptance | Home', () => {
-
-      cy.visit('/');
-
-      cy.get('[data-testid="account-2"]').click();
+      navigateToHome();
       cy.get('[data-testid="send-button"]').click();
       cy.url().should('eq', `${baseUrl}/${account.nickname}/transfer-coins`);
 
       cy.get('[data-testid="send-coins"]').should('be.visible');
       cy.get('[data-testid="receive-coins"]').should('not.be.visible');
     });
 
-    it('should land on receive page when receive CTA is clicked', () => {
+    it('should navigate to /transfer-coins when accessing it by the side-menu', () => {
       const account = mockedAccounts.at(2);
 
-      cy.visit('/');
+      navigateToHome();
+      cy.get('[data-testid="side-menu"]').click();
+      cy.get('[data-testid="side-menu-sendreceive-icon"]').click();
+      cy.url().should('eq', `${baseUrl}/${account.nickname}/transfer-coins`);
+    });
 
-      cy.get('[data-testid="account-2"]').click();
-      cy.get('[data-testid="receive-button"]').click();
+    it('should render balance and amount should equal account balance', () => {
+      const account = mockedAccounts.at(2);
+
+      navigateToTransfercoins(2);
+
+      cy.get('[data-testid="balance-amount"]').should(
+        'contain',
+        setAccountBalance(account),
+      );
+    });
+
+    it('should send coins to address', () => {
+      const account = mockedAccounts.at(2);
+      const recipientAccount = mockedAccounts.at(1);
+
+      const amount = 1000.12311132;
+
+      const standardFees = '1000';
+
+      navigateToTransfercoins(2);
+      cy.get('[data-testid="currency-field"')
+        .type(amount)
+        .should('have.value', customFormatNumber(amount));
+
+      cy.get('[data-testid="input-field"').type(recipientAccount.address);
+
+      cy.get('[data-testid="button"]').contains('Send').click();
       cy.url().should('eq', `${baseUrl}/${account.nickname}/transfer-coins`);
+      cy.get('[data-testid="send-confirmation"]').should('be.visible');
+
+      cy.get('[data-testid="send-confirmation-info"]').should(
+        'contain',
+        `Amount (${customFormatNumber(amount)}) + fees (${standardFees} nMAS)`,
+      );
+
+      cy.get('[data-testid="balance-amount"]').contains(formatStandard(amount));
 
-      cy.get('[data-testid="send-coins"]').should('not.be.visible');
-      cy.get('[data-testid="receive-coins"]').should('be.visible');
+      cy.get('[data-testid="send-confirmation-recipient"]').contains(
+        maskAddress(recipientAccount.address),
+      );
+
+      cy.get('[data-testid="button"]')
+        .contains('Confirm and sign with password')
+        .click();
+      cy.url().should('eq', `${baseUrl}/${account.nickname}/home`);
+    });
+
+    it('should process any % as input', () => {
+      const account = mockedAccounts.at(1);
+      const recipientAccount = mockedAccounts.at(2);
+      const standardFees = '1000';
+
+      navigateToTransfercoins(1);
+
+      // we have to wait or else balance is not initialized properly
+      cy.wait(1000);
```

Comment:
Use `.then()`. to act after something. The `wait()` not ensure what you want here.

Reply:
You have a nice read here if you want to believe me: 😄 

https://docs.cypress.io/guides/references/best-practices#Unnecessary-Waiting

Reply:
Totally ! I modified it :)